chalicelib/checks/higlass_checks.py

```diff
@@ -271,39 +249,42 @@ def patch_files_for_higlass_viewconf(connection, **kwargs):
             if kwargs['file_accession'] and file_accession != kwargs['file_accession']:
```

Comment:
This is handled above, on line 175 `if kwargs['file_accession']:` ...
Doesn't hurt but may be best to remove code that does duplicate things